package.json

```diff
@@ -50,9 +50,13 @@
         "xmldoc": "^0.1.2",
         "grunt-cleanempty": "1.0.3"
     },
+    "devDependencies": {
+        "http-server": "^0.8.0"
+    },
     "scripts": {
         "postinstall": "grunt install",
-        "test": "grunt test && grunt build-browser-compressed"
+        "test": "grunt test && grunt build-browser-compressed",
+        "start": "http-server --port=8000 src"
```

Comment:
Can you please change `src` to `.` here, so that we can run `src/` and `dist/` with the same server?  Thimble, and the docs above, assume this.

After that, this is good to go.